pkg/packagekit/autoupdates.jsx

```diff
@@ -35,7 +35,7 @@ import { useInit } from "hooks";
 const _ = cockpit.gettext;
 
 function debug() {
-    if (window.debugging == "all" || window.debugging == "packagekit")
+    if (window.debugging == "all" || window.debugging?.includes("packagekit"))
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18492-20230313-062223-0ed5c399-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)